### PR TITLE
Update template aa1 to master

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [x] Documentation (non-breaking change which adds documentation)
 - [x] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently our coding style is disgusting and not up to our 2020/08 standards.
- Currently the contents of the README.md and other GitHub related files is outdated.
- Currently this repository has no maintainer.

**What is the new behavior (if this is a feature change)?**
- The coding style has been improved tremendously.
- The contents of the README.md and other GitHub related files has been improved.
- @Aichnerc has been set as the maintainer of this repository.

**Other information**:
- The version has been updated to `v2.0.0`.

### CODE REVIEW

**Review by Theralun — APPROVED** [review]
Cloudy with a chance of regions.

File: README.md [hunk 1]
```diff
@@ -27,13 +25,10 @@
...
 
 <a href="https://liberapay.com/aichner/donate"><img alt="Donate using Liberapay" src="https://liberapay.com/assets/widgets/donate.svg"></a>
-<a href="https://archiveprogram.github.com/" target="_blank" rel="noopener noreferrer">
-  <img src="https://www.aichner-christian.com/img/logo/code_vault.jpg" style="margin-top: 0.5rem" alt="Mars Time Logo" height="30">
-</a>
-
 
 ## Table of contents
 
+- [Table of contents](#table-of-contents)
```
Comment (pinterid) [inline README.md:31]: Is Codacy ignoring this file?

**Review by pinterid — APPROVED** [review]
LGTM

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -12,34 +12,33 @@
...
-- Each pull request must answer one more more issues
-- Each pull request must have at least one reviewer
-- Each pull request must have at least one assigned developers
-- Each pull request must have a label
-- Each pull request should be merged by a reviewer
-
+-   Each pull request should only follow one topic
+-   You must follow the guidelines / template
+-   Each pull request must have at least one reviewer
+-   Each pull request must have at least one assigned developers
```
Comment (kleberbaum) [inline CONTRIBUTING.md]:
```suggestion
-   Each pull request must have at least one assigned developer
```

**Review by kleberbaum — CHANGES_REQUESTED** [review]
Should be ok to merge after a few changes.

I do also think this title should be in title case.
image

File: CONTRIBUTING.md (hunk 2 above)
Comment (kleberbaum) [reply CONTRIBUTING.md]: I have already dealt with the same issues in my snek repos.

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -1,12 +1,13 @@
 <p align="center">
   <a href="https://www.aichner-christian.com/" target="_blank" rel="noopener noreferrer">
-    <img src="https://www.aichner-christian.com/img/logo/logo_web.png" alt="Agency Logo" height="150">
+    <img src="https://avatars1.githubusercontent.com/u/50574311?s=200&v=4g" alt="Agency Logo" height="150">
   </a>
 </p>
 
 <h3 align="center">Official code of conflict</h3>
```
Comment (kleberbaum) [inline CODE_OF_CONDUCT.md]:
```suggestion
<h3 align="center">Official Code of Conflict</h3>
```

**Review by kleberbaum — CHANGES_REQUESTED** [review]
Almost forgot about this one.

File: CONTRIBUTING.md (hunk 2 above)
Comment (Aichnerc) [reply CONTRIBUTING.md]: I copied it from `snek-at/ops-front`.

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Pull request merged**